hpobench/util/data_manager.py

```diff
@@ -537,7 +539,6 @@ def __init__(self, data_path: Union[str, Path]):
                             lock_path=f'{hpobench.config_file.cache_dir}/lock_nasbench_101_data', delay=0.5)
     def _download(self):
         from tqdm import tqdm
```

Comment:
I am fine with adding tqdm as a dependency, but then it should be part of requirements.txt and not only required for paramnet?

Reply:
hm, the thing is, we only use it for the paramnet benchmark (more precisely in the paramnet dataloader). The Hpobench package does not rely on it. But if you prefer it more in the general requirements, that's no problem at all :-)

Reply:
But this is not the paramnet dataloader, this is NASBench_101. The paramnet dataloader does not make use of tqdm. Okay, but then, since it is only one dataloader, I guess it's best to move the dependency to the respective benchmark requirements.

Reply:
ouh, that's a point. I forgot the tqdm requirement for the nasbench101 benchmark. But the paramnet data loader also uses tqdm. (line 628)